tests/Unit/inc/Engine/Optimization/Minify/CSS/Combine/optimize.php

```diff
@@ -63,5 +63,8 @@ public function testShouldCombineCSS( $original, $combined, $cdn_host, $cdn_url,
 			$combined,
 			$this->combine->optimize( $original )
 		);
+
+		$this->assertTrue( $this->filesystem->exists( 'wordpress/wp-content/cache/min/1/0bca286481748a69cdbe9a6695015ec9.css' ) );
```

Comment:
Move CSS files in fixtures and use a loop to parse those and make the assertTrue. What do you think?